molgenis-security/src/main/java/org/molgenis/security/twofactor/model/UserSecretFactory.java

```diff
@@ -6,8 +6,9 @@
 
 @Component
 public class UserSecretFactory
-    extends AbstractSystemEntityFactory<UserSecret, UserSecretMetaData, String> {
-  UserSecretFactory(UserSecretMetaData userSecretMetaData, EntityPopulator entityPopulator) {
+    extends AbstractSystemEntityFactory<UserSecret, UserSecretMetadata, String> {
+
```

Comment:
Assuming this is the result of a "refactor" action I'm not sure how this newline ended up here, but I think we want to remove it right?